src/command-testing.js

```diff
@@ -94,23 +127,53 @@ window.addEventListener('keyup', (event) => {
                 // const pic1Path = new URL(pic1RelPath, url).href;
 
                 // const absolutePath = __dirname + pic1RelPath;
-                var pic1 = document.createElement('img');
-                pic1.onload = function() {
-                    console.log('Image loaded:', this.src);
-                };
-                pic1.onerror = function() {
-                    console.error('Error loading image:', this.src);
-                };
-                pic1.src = `/src/preview-pics/sampleImage.png`;
 
+                // var pic2 = createImage(previewPic2, 100, 100, "10px");
+                // var pic3 = createImage(previewPic3, 100, 100, "10px");
+                // var pic4 = createImage(previewPic4, 100, 100, "10px");
+                // var pic5 = createImage(previewPic5, 100, 100, "10px");
+
+                // text_div.appendChild(text);
+                // text_div.appendChild(pic2);
+                // text_div.appendChild(pic3);
+                // text_div.appendChild(pic4);
+                // text_div.appendChild(pic5);
+
+                // Appends any number of images to popup UI depending on number of imageUrls passed
+                // Currently displays in 100x100 per image TBD: if aspect ratio not defined enough
                 text_div.appendChild(text);
-                text_div.appendChild(pic1);
+                for (var i = 0; i < imageUrls.length; i++) {
+                    var img = createImage(imageUrls[i], 100, 100, "10px");
+                    text_div.appendChild(img);
+                }
+
+                text_div.style.whiteSpace = 'nowrap';
+
 
                 popup.appendChild(text_div);
                 document.body.appendChild(popup);
             } else {
+                // This is the branch for when 'Q' is clicked multiple times as 'CTRL' is held
                 var preview = document.getElementById(previewId);
-                preview.children[0].innerHTML = "Switch tab num times: " + numSwitches % 5;
+
+                //querySelectorAll() gets all images from preview box
+                var images = preview.querySelectorAll('img');
+                var numImages = images.length;
+
+                // Updates the outline to highlight the active image:
+                // numSwitches: number of times Q is clicked while CNTRL is held down
+                // numImages: number of images in preview box
+                var currentIndex = numSwitches % numImages;
+
+                 // Update the 3 pixel thick blue outline to highlight the active image
+                images.forEach((image, index) => {
+                    if (index === currentIndex) {
+                      image.style.outline = '3px solid blue';
```

Comment:
@riteshsb I think this is the code that you can just factor out and call again at around line 146, to make sure the outline appears around the second box upon initialization.